inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/strided_slice.hpp

```diff
@@ -44,4 +45,29 @@ class StridedSliceLayerTest : public testing::WithParamInterface<StridedSlicePar
 protected:
     void SetUp() override;
 };
+
+struct Slice8SpecificParams {
```

Comment:
There are already shared test class for [Slice](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/slice.hpp). Create new file and use it please

Reply:
Done.